projects/kit/src/lib/masks/number/utils/tests/to-number-parts.spec.ts

```diff
@@ -0,0 +1,67 @@
+import {toNumberParts} from '../to-number-parts';
+
+describe('toNumberParts', () => {
```

Comment:
Please, add tests for other decimal separators

<img width="725" alt="image" src="https://github.com/taiga-family/maskito/assets/12021443/f4c496a6-2ff9-4c8a-89b2-6cdd8701cd13">

Reply:
✅